lib/list.js

```diff
@@ -30,10 +30,28 @@ module.exports = function (filter, callback) {
     db.find({date: Number(date.format('YYYYMMDD'))}).sort({ datetime: 1 }).exec(function (err, registers) {
       if (err) return callback(err)
       var totalTime = 0
+      var totalTimeObj = {}
+      var tags = []
       for (var i in registers) {
-        totalTime += registers[i].time
+        var itemTime = registers[i].time
+        totalTime += itemTime
+        registers[i].tags.map(item => {
```

Comment:
Please check this, I think is out `undefined`

Reply:
Fixed